README.md

```diff
@@ -129,9 +129,14 @@ strategy:
 ```
 
 ## Setting up CLion 
+
+### Opening project
 Configuring CLion with the extension template requires a little work. Firstly, make sure that the DuckDB submodule is available. 
 Then make sure to open `./duckdb/CMakeLists.txt` (so not the top level `CMakeLists.txt` file from this repo) as a project in CLion.
 Now to fix your project path go to `tools->CMake->Change Project Root`([docs](https://www.jetbrains.com/help/clion/change-project-root-directory.html)) to set the project root to the root dir of this repo.
 
-Now to configure the build targets, copy the CMake variables specified in the Makefile and ensure
-the build directory is set to `../build/<build_mode>`.
\ No newline at end of file
+
+### Debugging
+To set up debugging in CLion, there are two simple steps required. Firstly, in `CLion -> Settings -> Build, Execution, Deploy -> CMake` you will need to add the desired builds (e.g. Debug, Release, RelDebug, etc). There's different ways to configure this, but the easiest is to leave all empty, except the `build path`, which needs to be set to `../build/{build type}`. Now on a clean repository you will first need to run `make {build type}` to initialize the CMake build directory. After running make, you will be able to (re)build from CLion by using the build target we just created.
```

Comment:
it should be `CLion Preferences`

Reply:
Its called Settings on my version of OSX Clion. I've added preferences it for clarity!